app/front-end/src/features/editor/components/editorView/editorColumnMenu.tsx

```diff
@@ -1,12 +1,20 @@
 import { EditorColumnMenuAggregationItem, EditorColumnMenuSortItem } from '@/features/editor/components/editorView';
 import { useWorkspaceContext } from '@/features/editor/hooks';
 import { FileContentAggregationActions, SortEnum } from '@/features/editor/types';
+import { Colors } from '@/types';
 import { Divider } from '@mui/material';
 import { styled } from '@mui/material/styles';
 import { GridColumnMenuContainer, GridColumnMenuHideItem, GridColumnMenuProps } from '@mui/x-data-grid';
 
 const StyledGridColumnMenuContainer = styled(GridColumnMenuContainer)(({ theme }) => ({
-  backgroundColor: theme.palette.secondary.main,
+  borderRadius: '1rem',
+  backgroundColor: theme.palette.mode === 'light' ? Colors.backgroundPrimaryLight : Colors.backgroundPrimaryDark,
```

Comment:
for `backgroundColor` use `theme.palette.background.default`. In theme context its defined like this:
```
...
    background: {
        default: mode === 'light' ? Colors.backgroundPrimaryLight : Colors.backgroundPrimaryDark,
        paper: mode === 'light' ? Colors.backgroundSecondaryLight : Colors.backgroundSecondaryDark,
    },
...
```